subsys/logging/backends/log_backend_ble.c

```diff
@@ -1,210 +0,0 @@
-/*
```

Comment:
I don't agree with removing the log backend for BLE, backends are not dependent on UART by design, they use a way simpler pipe like read/write API, especially compared to injecting a BLE to UART emulator into the chain, so it now looks like BLE -> UART -> shell uart backend, instead of just shell ble backend.

Reply:
Thanks for your feedback, @bjarki-trackunit!

TLDR: I don't have a strong preference on this and we can take it out of the PR if there are disagreements :-).

I agree that is simpler using directly a subsystem backend such as Logging vs abstracting it behind an UART device. I, myself, submitted a PR not so long ago attempting to add Bluetooth LE as a Shell backend (along the same order of ideas). This PR, for various reasons didn't make it upstream (see discussion on #68076).

Taking the feedback from that previous PR: The idea on this one is to go for a common driver that deals directly with the Bluetooth LE GATT Service and interactions, and by doing that, make it easier to maintain the code while serving the same purposes. This approach is inspired by what USB CDC ACM does which implements the device class to act as an UART driver vs implementing on a per-subsystem basis.

On a semi-related data-point: Interestingly enough, the BLE Logging backend was conceived in the context of needing UART over Bluetooth LE, but not implemented due to the author not feeling comfortable dealing with the BLE Stack on such a broad surface. Here's the comment where it originated: https://github.com/zephyrproject-rtos/zephyr/issues/34949#issuecomment-1486449043

In the end, I'm in pro of:
1. Making the feature (Console, Shell, Logging) easily accessible and intuitive to use; which can be achieved by good documentation, consistency, among other things.
2. Making it easier for maintainers to keep code updated and bug-free.

As a result, I consider that maintaining the driver is more sustainable than individual backends.

Nevertheless, if after all the context behind, there are still disagreements on this point, I can take it out of scope and discuss it as a separate issue/PR. Would you agree, @bjarki-trackunit ?